docs/ko/docs/tutorial/security/first-steps.md

```diff
@@ -0,0 +1,183 @@
+# 보안 - 첫 단계
+
+어떤 도메인에 **백엔드** API가 있다고 상상해봅시다.
+
+그리고 또다른 도메인 또는 동일한 도메인의 다른 경로에 (또는 모바일 애플리케이션에) **프론트엔드**가 있다고 해봅시다.
+
+그리고 **사용자이름** 및 **비밀번호**를 사용하여, 프론트엔드가 백엔드로 인증하는 방법을 원합니다.
+
+**OAuth2**를 사용하여 그것을 **FastAPI**로 구축할 수 있습니다.
+
+그러나 필요한 작은 정보를 찾기 위해 긴 명세서 전체를 읽는 시간을 절약할 수 있습니다.
+
+보안을 처리하기 위해 **FastAPI**가 제공하는 도구를 사용해봅시다.
+
+## 생김새
+
+우선 그냥 코드를 사용하여 이것이 어떻게 작동하는 지 보고, 어떤 일이 발생했는 지 이해하기 위해 나중에 돌아옵시다.
+
+## `main.py` 생성
+
+`main.py` 파일에 예시를 복사합니다:
+
+```Python
+{!../../../docs_src/security/tutorial001.py!}
+```
+
+## 실행
+
+!!! info "정보"
+    우선 <a href="https://andrew-d.github.io/python-multipart/" class="external-link" target="_blank">`python-multipart`</a>를 설치합니다.
+
+    예를 들어 `pip install python-multipart`와 같습니다.
+
+    이것은 **OAuth2**가 `username` 및 `password`를 전송하는 데 "폼 데이터"를 사용하기 때문입니다.
+
+
+아래의 방법을 사용하여 예시를 실행합니다:
+
+<div class="termy">
+
+```console
+$ uvicorn main:app --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+## 확인
+
+대화형 문서로 이동합니다: <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>.
+
+아래와 같은 것을 볼 수 있습니다:
+
+<img src="/img/tutorial/security/image01.png">
+
+!!! check "Authorize 버튼!"
+    이미 반짝이는 새로운 "Authorize" 버튼이 있습니다.
+
+    그리고 *경로 작동*에는 클릭할 수 있는 작은 자물쇠가 우측-상단 모서리에 있습니다.
+
+그리고 만약 이것을 클릭하면, `username` 및 `password` (그리고 다른 선택적 필드) 를 입력하는 작은 인가 폼이 뜹니다:
```

Comment:
How about using "허가" instead of "인가"? 
maybe it has almost same mean, but it's hard to understand "인가" intuitively!
Thank you! tell me about your opinion 😊

Reply:
I translated as `인가` because of the word `Authorization`. I think it usually translate as `인가` in a security part (like the word `Authentication` as `인증`)!

Reply:
If you check [인증과 인가 구글 검색](https://www.google.com/search?q=%EC%9D%B8%EC%A6%9D%EA%B3%BC+%EC%9D%B8%EA%B0%80&sxsrf=AOaemvLfqx-0VGwGlBhqAh_wxSu2-N8OfQ%3A1635927848566&ei=KEeCYcb7IbLN5OUPvpqo6Aw&oq=%EC%9D%B8%EC%A6%9D%EA%B3%BC+%EC%9D%B8%EA%B0%80&gs_lcp=Cgdnd3Mtd2l6EAMyBQgAEIAEMgUIABCABDIFCAAQgAQ6CAgAEIAEELADOgkIABCwAxAHEB46BAgjECc6CwgAEIAEELEDEIMBOhEILhCABBCxAxCDARDHARDRAzoLCC4QgAQQxwEQ0QM6CAgAEIAEELEDOgQIABAeSgQIQRgBUJ9DWP1LYMFNaARwAHgCgAHBAogBwxOSAQgwLjExLjIuMZgBAKABAcgBCsABAQ&sclient=gws-wiz&ved=0ahUKEwjGr6qS4vvzAhWyJrkGHT4NCs0Q4dUDCA4&uact=5), there are lots of docs or posts that translate `Authorization` as `인가` and `Authentication` as `인증` for explain of security.

Reply:
I didn't consider using "인가" with`Authorization` 😅 
Thank you for giving your opinion! i'll also check other PR too!

Reply:
@DevDae 

Can you approve the review, if you think this translation is good to merge?